src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -824,7 +826,14 @@ public World get(final CommandEvent e) {
 				return e.getSender() instanceof Player ? ((Player) e.getSender()).getWorld() : null;
 			}
 		}, 0);
-		
+		//UnknownCommandEvent
+		if(Skript.classExists("org.bukkit.event.command.UnknownCommandEvent")) EventValues.registerEventValue(UnknownCommandEvent.class, CommandSender.class, new Getter<CommandSender, UnknownCommandEvent>() {
```

Comment:
You need to moEventValues.registerEventValue(UnknownCommandEvent.class, CommandSender.class, new Getter<CommandSender, UnknownCommandEvent>()`` onto the next line

Reply:
Done